golint/golint.go

```diff
@@ -47,10 +47,14 @@ func main() {
 		// checks are run. It is no valid to mix target types.
 		var dirsRun, filesRun, pkgsRun int
 		var args []string
-		for _, arg := range flag.Args() {
-			if strings.HasSuffix(arg, "/...") && isDir(arg[:len(arg)-len("/...")]) {
+		for _, arg := range flag.Args()  {
+			basePath := arg[:len(arg)-len("/...")]
```

Comment:
There's no guarantee that arg has "/..." as a suffix, nor even that it has length at least four, so this operation will panic for shorter names such as "fmt". Use this pattern to remove an optional suffix:
```
if y := strings.TrimSuffix(x, "/..."); y != x { ... }
```

Reply:
Thanks! I've fixed it according to your suggestion. Could you please check it out again?